build/launchWebTest.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const path = require('path');
+const test_web = require('@vscode/test-web');
+async function go() {
```

Comment:
With this, we don't need to compile the ts file anymore, we can just run webpack and that's it.

Reply:
Besides TS is smart enough to give decent completions for this.